force-app/main/default/profiles/Sales Director %E2%80%93 APAC.profile-meta.xml

```diff
@@ -0,0 +1,329 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Profile xmlns="http://soap.sforce.com/2006/04/metadata">
+    <custom>true</custom>
+    <fieldPermissions>
+        <editable>true</editable>
+        <field>Lead.Address</field>
+        <readable>true</readable>
+    </fieldPermissions>
+    <fieldPermissions>
```

Comment:
is it right permission according to ex5.1

Reply:
changed